src/language-server/__tests__/fixtures/documents/commentWithTemplate.ts

```diff
@@ -0,0 +1,41 @@
+// hint for reading the snapshots generated by this file:
+// it is important to check that a rule doesn't only enter `meta.embedded.block.graphql`
+// but also leaves it again
+
+/* NormalComment */ `query Foo {test}`;
+
+// prettier-ignore
+{
+const _1 = /* GraphQL */`query Q1 {test}`;
+const _2 =
+/* GraphQL */`
+query Q2 {
+  test
+}
+`;
+}
+
+/* GraphQL */ `query Q3 {test}`;
+/*GraphQL*/ `query Q4 {test}`;
+/** GraphQL */ `query Q5 {test}`;
+
+const _3 = /* GraphQL */ `
+  query Q6 {
+    test
+  }
+`;
+
+// syntax highlighting cannot work in all examples after this - textmate grammars don't work over multiple lines like this
```

Comment:
While we cannot syntax highlight anything after that line, the other features will still work. We *could* disable them with stricter regular expressions, but I'm not sure if we should.